DashKitUI/View Models/DashSettingsViewModel.swift

```diff
@@ -104,6 +89,23 @@ class DashSettingsViewModel: DashSettingsViewModelProtocol {
             }
         }
     }
+    
+    func suspendDelivery(duration: TimeInterval) {
+        guard let reminder = try? StopProgramReminder(value: duration) else {
+            assertionFailure("Invalid StopProgramReminder duration of \(duration)")
+            return
+        }
+
+        pumpManager.suspendDelivery(withReminder: reminder) { (error) in
+            
```

Comment:
[nit] Do you want a TODO here as well for displaying the error, if that work is for a separate PR?

Reply:
Separate PR: https://tidepool.atlassian.net/browse/LOOP-1176